lib/g5_authenticatable/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module G5Authenticatable
-  VERSION = '1.1.4'
+  VERSION = '1.1.5-1'
```

Comment:
is this a pre version?

Reply:
It currently is...thinking we should push out a minor release when this is accepted?

Reply:
Yeah we can build 1.1.5 or 1.2.0 when accepted - i'm leaning towards the minor level bump as well